src/shared/animations/index.js

```diff
@@ -0,0 +1,88 @@
+import welcome from './welcome.json';
+import language from './language.json';
+import ledger from './ledger.json';
+import onboardingComplete from './onboardingComplete.json';
+import get from 'lodash.get';
+
+import themes from '../themes/themes';
+
+export const animations = {
+    language,
+    ledger,
+    onboardingComplete,
+    welcome,
+};
+
+/**
+ * Returns colorised Lottie animation data
+ * @param {string} animationName - Target animation name
+ * @param {object} themeName - Target colors
+ * @returns {object} -Target animation data
```

Comment:
```suggestion
 * @returns {object} - Target animation data
```
This really bothered me for some reason 😂 

![image](https://user-images.githubusercontent.com/19519564/56857332-1b62f300-693a-11e9-9418-2442e05c56d4.png)